CMakeLists.txt

```diff
@@ -1,8 +1,14 @@
-cmake_minimum_required(VERSION 2.8)
-set(CMAKE_SYSTEM_NAME Linux)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Ubuntu supports cmake 3.5 in the official repositories. What is the reason behind 3.5? We need to upgrade all the machines with RIVER for this change otherwise.

Reply:
Removed the requirement for cmake 3.7. I strongly suggest setting a DESTDIR when running cmake since the default DESTDIR for Windows is %programfiles%.